apps/idam/idam-api/idam-api.yaml

```diff
@@ -7,15 +7,14 @@ spec:
   releaseName: idam-api
   values:
     java:
-      image: hmctspublic.azurecr.io/idam/api:prod-97407f6-20210803165122
+      image: hmctspublic.azurecr.io/idam/api:prod-81ccbf4-20220504141739
       replicas: 4
       ingressHost: idam-api.platform.hmcts.net
       aadIdentityName: idam
       useInterpodAntiAffinity: true
       environment:
+        idam.scheduled.batch-processing.enabled: false
         ENDPOINTS_INFO_ENABLED: true
-        CACHING_USERS_MAXTHREADS: 3
```

Comment:
Removing these variables (no longer used) should trigger deployment by the kustomize controller 👍